index.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2019-present, ProductBoard, Inc.
+ * All rights reserved.
+ */
+
+const fs = require("fs");
+const child = require("child_process");
+const gzipSize = require("gzip-size");
```

Comment:
We're using single quotes where possible.

--
I am sorry that linting isn't exactly correctly set up in the repo, but can you please unify the syntax in the PR?

Reply:
Yep, I'm going to fix it, my bad!